### ISSUE TITLE
Warning: Option Z Is Unstable

### ISSUE THREAD

**Issue by sharpjs** [rsolomo/cargo-check#12]
I've been getting this warning on every invocation of `cargo check` on stable rust (1.10):

```
$ cargo check
   Compiling ...redacted...
warning: the option `Z` is unstable and should only be used on the nightly compiler, but it is currently accepted for backwards compatibility; this will soon change, see issue #31847 for more details
```

**Comment by AndrewBrinker** [comment]
This seems to still be happening. Is there some way to get no-trans on stable without using `-Z`?
